certification/internal/engine/engine.go

```diff
@@ -51,12 +52,21 @@ type CraneEngine struct {
 func (c *CraneEngine) ExecuteChecks() error {
 	log.Debug("target image: ", c.Image)
 
-	// prepare crane runtime options, if necessary
-	options := make([]crane.Option, 0)
+	// prepare crane runtime options for NewTag call, if necessary
+	tagOptions := make([]name.Option, 0)
+
+	log.Debug("getting tag info for image")
+	tag, err := name.NewTag(c.Image, tagOptions...)
```

Comment:
If you don't ever anticipate us putting options there, just leave it off. I don't think we would need options here.